.editorconfig

```diff
@@ -67,4 +67,12 @@ csharp_style_pattern_matching_over_is_with_cast_check = true:suggestion
 csharp_style_pattern_matching_over_as_with_null_check = true:suggestion
 csharp_style_inlined_variable_declaration = true:suggestion
 csharp_style_throw_expression = true:suggestion
-csharp_style_conditional_delegate_call = true:suggestion
\ No newline at end of file
+csharp_style_conditional_delegate_call = true:suggestion
```

Comment:
Please make a separate PR - also tag @kuhlenh and @jmarolf since we're considering streamlining the set of formatting options we expose based on telemetry about what is actually changed by users.